CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Added
 * Accordion: Nieuwe variant "compact-black". ([#2214](https://github.com/dso-toolkit/dso-toolkit/issues/2214))
 * Button: Tertiary button uitbreiden met truncate modifier. ([#2140](https://github.com/dso-toolkit/dso-toolkit/issues/2140))
+* Unordered List: Iconen (status-warning en forbidden) kunnen weergeven ipv bullet ([#2274](https://github.com/dso-toolkit/dso-toolkit/issues/2274))
```

Comment:
De naam van het component is "List"

Reply:
Moet wel ergens tijdelijk zijn dat het alleen om ul gaat.

Reply:
Ja leuk dat je de variant tussen haakjes hebt gezet, goed idee!